src/Plugin/VoteUpDownWidgetBase.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Drupal\vud\Plugin;
+
+use Drupal\Component\Plugin\PluginBase;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+
+/**
+ * @inheritdoc
+ */
+abstract class VoteUpDownWidgetBase extends PluginBase implements VoteUpDownWidgetInterface {
```

Comment:
I mentioned before:
> Please use `{@inheritdoc}` on methods from classes that implement an interface, and they do not need any extra documentation. Please read details on [drupal standards about inheritdoc](https://www.drupal.org/docs/develop/coding-standards/api-documentation-and-comment-standards#inheritdoc).

That's still pending.

I guess I was not clear. I did not mean to say that you should use inheritdoc on the class docblock, what I tried to say is that you should use that everywhere where you are not changing the purpose of an inherited method, especially on methods which the main documentation on the interface should be the same than in the class implementation.